src/components/LegacyPortal.js

```diff
@@ -0,0 +1,59 @@
+// This file is a fallback for a consumer who is not yet on React 16
+// as createPortal was introduced in React 16
+// Extracted from https://github.com/tajo/react-portal
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+
+export default class Portal extends React.Component {
```

Comment:
please import `Component` in the same line we import React 
(`import React, { Component } from 'react'`) so we just do `extends Component`